### ISSUE TITLE
Feature: Add OVH to the supported Provider

### ISSUE THREAD

**Issue by DrummyFloyd** [berops/claudie#1067]
### Motivation

I think to an another European Provider such a OVH would be a great features 

### Description

[OVH Kubernetes](https://www.ovhcloud.com/fr/public-cloud/kubernetes/?at_medium=sl&at_platform=google&at_campaign=AdWords&at_creation=noi_ovh_fr_se_enterprise_publiccloud_defensive(Kubernetes)&at_variant=676696754575&at_network=search&at_term=ovh%20kubernetes&gclid=CjwKCAjwp8OpBhAFEiwAG7NaEq_JgiahEg2RY_TJRWRrKd6_qUXLv8xFa2BZm_6GtxTXUwNlifQ4ahoCYOoQAvD_BwE)

### Exit criteria

Example:
- [ ] Implement the feature
- [ ] Add feature to the Claudie

**Comment by Utopiah** [comment]
Indeed, commenting to stay updated on that but having it in addition to Hetzner would be great, especially as their Kubernetes solution already works great (you get a `kubeconfig.yml` file in minutes via their web interface).

**Comment by bernardhalas** [comment]
Hi @Rohithzr , before expanding the provider set, we would like to focus on the usability and stability improvements.
That being said, we'd be happy to review your PR if you decide to raise one.
